Create-DefaultListForms.ps1

```diff
@@ -169,27 +180,32 @@ if ($null -eq $csvRows) {
     break
 }
 
-foreach ($row in $csvRows) 
+foreach ($row in $csvRows)
 {
     Write-Output "[$($row.siteUrl)] [$($row.listTitle)]"
 
-    Connect-PnPOnline -Url $row.siteUrl
+    if($UseWebLogin) {
+        Connect-PnPOnline -Url $row.siteUrl -UseWebLogin
+    }
+    else {
+        Connect-PnPOnline -Url $row.siteUrl
+    }
```

Comment:
This is the main portion of the changes (sorry for all the formatting changes on top of this)